### PR TITLE
[F] LRAC-13423 Use Loading indicator from Clay instead AC spinner

### PR DESCRIPTION
**Pull request by JLucasM**
LRAC-13423: [Jira Issue](https://issues.liferay.com/browse/LRAC-13423)

### CODE REVIEW

**Comment by JLucasM** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/VerticalTimeline.tsx [hunk 1]
```diff
@@ -337,7 +337,12 @@ const VerticalTimeline: FC<IVerticalTimelineProps> = ({
 	timeZoneId
 }) =>
 	loading ? (
-		<Spinner alignCenter={false} className='flex-grow-1' spacer />
+		<ClayLoadingIndicator
+			className={getCN('flex-grow-1 spinner-root', {
+				['spinner-center']: false,
+				['spinner-spacer']: !false
+			})}
+		/>
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/VerticalTimeline.tsx:345]: It should be:

```suggestion
		<ClayLoadingIndicator
			className='flex-grow-1 spinner-spacer spinner-root'
		/>
```

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/VerticalTimelineDeprecated.tsx [hunk 2]
```diff
@@ -200,7 +200,14 @@ const VerticalTimeline: FC<IVerticalTimelineProps> = ({
...
 
 	if (loading) {
-		return <Spinner alignCenter={false} className='flex-grow-1' spacer />;
+		return (
+			<ClayLoadingIndicator
+				className={getCN('flex-grow-1 spinner-root', {
+					['spinner-center']: false,
+					['spinner-spacer']: !false
+				})}
+			/>
```
Comment (interaminense) [inline modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/shared/components/VerticalTimelineDeprecated.tsx:209]: The same:

It should be:

```suggestion
		<ClayLoadingIndicator
			className='flex-grow-1 spinner-spacer spinner-root'
		/>
```

**Review by interaminense — CHANGES_REQUESTED** [review]
What's missing to do?

1. Delete spinner component
2. Delete spinner unit test
3. Delete spinner UIKit

Thanks a lot for you hard work!

**Comment by JLucasM** [comment]
ci:test:sf

**Comment by JLucasM** [comment]
ci:test:relevant

**Review by interaminense — APPROVED** [review]
LGTM! Thanks @JLucasM

**Comment by interaminense** [comment]
ci:forward

**Pull request closed without merging**